testcontainers/src/core/containers/request.rs

```diff
@@ -35,6 +35,7 @@ pub struct ContainerRequest<I: Image> {
     pub(crate) cgroupns_mode: Option<CgroupnsMode>,
     pub(crate) userns_mode: Option<String>,
     pub(crate) startup_timeout: Option<Duration>,
+    pub(crate) working_dir: Option<String>,
```

Comment:
Let's also add the field to `impl<I: Image + Debug> Debug for ContainerRequest<I> {`

Reply:
Thank you, I missed that one :)